error/index.html

```diff
@@ -1,16 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <meta http-equiv="X-UA-Compatible" content="ie=edge">
-        <title>Registration error</title>
-        <link rel="stylesheet" href="../style.css">
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <title>Holo Quickstart: Part II</title>
+      <link rel="stylesheet" href="style.css">
     </head>
-    <body>
-        <h1>Oops!</h1>
-	<p>Something went wrong with your email verification. Please go to <a href="https://quickstart.holo.host">https://quickstart.holo.host</a> and start the registration process again.</p>
-	<p>Error code <span id="error_code"></span>: <span id="error_text"></span></p>
-	<script src="index.js"></script>
```

Comment:
Depending on a type of error authorizer is capable of sending an error code in the url's query string: `/error?err=501`. index.js  can read this parameter and display meaningful error messages to user. It is not implemented in current version therefore we should remove index.js, but I think that in the future being able to display error messages to the user would be very helpful in troubleshooting.

Reply:
BTW it looks soooo goood! :-)